README.md

```diff
@@ -28,35 +28,37 @@ To build only the native library:
 
 ## Running
 ### Prerequisites
-Before running, you'll need to provide a pi data file. The file should be a **SINGLE LINE** text file containg the
+Before running, you'll need to provide a Pi data file. The file should be a **SINGLE LINE** text file containg the
 digits of pi. 
 
-The [file](src/main/resources/Pi1M.txt) provided with this repository is only 1M digits long, which isn't very 
+NOTE: The [file](src/main/resources/Pi1M.txt) provided with this repository is only 1M digits long, which isn't very 
 interesting to search. 
 
 #### Getting a data file
-You download some 1B+ digit calculations of pi from [here](https://stuff.mit.edu/afs/sipb/contrib/pi/), 
-or you can calculate your own pi data file with [y-cruncher](http://www.numberworld.org/y-cruncher/), or you
-
-Set the location of the file as an environment variable:
+To get Pi data, you can do the following: 
+- Download 1B+ digit calculations of Pi from [here](https://stuff.mit.edu/afs/sipb/contrib/pi/)
+- Calculate your own Pi data file with [y-cruncher](http://www.numberworld.org/y-cruncher/)
+- Perform the following steps:
+  
+1. Set the location of the file as an environment variable:
```

Comment:
I like the update to use an unordered list for the data file options. This line about setting the location of the file starts the next section, which should be titled "Configuring the data file" or "using your data file" or something like that